static/app/views/discover/table/tableActions.tsx

```diff
@@ -148,18 +151,25 @@ function FeatureWrapper(props: FeatureWrapperProps) {
   );
 }
 
-function HeaderActions(props: Props) {
+function TableActions(props: Props) {
+  const investigationRuleProps = {...props, numSamples: props.tableData?.data?.length};
   return (
-    <Fragment>
-      <FeatureWrapper {...props} key="edit">
-        {renderEditButton}
-      </FeatureWrapper>
-      <FeatureWrapper {...props} key="download">
-        {renderDownloadButton}
-      </FeatureWrapper>
-      {renderSummaryButton(props)}
-    </Fragment>
+    <div className="table-actions">
+      <Fragment>
```

Comment:
Do we need the className?
I believe Fragment can be removed now that you wrapped it in div.